src/mlpack/methods/xgboost/loss_functions/sse_loss.hpp

```diff
@@ -86,22 +81,85 @@ class SSELoss
   template<bool UseWeights, typename MatType, typename WeightVecType>
   double Evaluate(const MatType& input, const WeightVecType& /* weights */)
   {
```

Comment:
Since `weights` is ignored, should we issue a `static_assert` error or something if `UseWeights` is true since `SSELoss` doesn't support it?